hudi-common/src/main/java/org/apache/hudi/common/model/PartitionBucketIndexHashingConfig.java

```diff
@@ -196,24 +196,51 @@ public static Option<PartitionBucketIndexHashingConfig> loadHashingConfig(Hoodie
 
   /**
    * Get Latest committed hashing config instant to load.
+   * If instant is empty, then return latest hashing config instant
    */
-  public static String getLatestHashingConfigInstantToLoad(HoodieTableMetaClient metaClient) {
+  public static Option<String> getHashingConfigInstantToLoad(HoodieTableMetaClient metaClient, Option<String> instant) {
     try {
       List<String> allCommittedHashingConfig = getCommittedHashingConfigInstants(metaClient);
```

Comment:
Can we assume there is always config files once the partition bucket index is enabled?

Reply:
Yes we can.
Currently, we have two methods to use partition-level bucket indexing:

1. Enabling during table creation via DDL: When enabled during table creation, this method initializes a 0000000000000.hashing_config file through the catalog. 

2. Upgrading existing table-level bucket indexes via the CALL command: For tables that already use table-level bucket indexing, invoking a CALL command triggers an upgrade process. This generates a replace-commit instant  and initializes a corresponding <replace-commit>.hashing_config file